mlflow/go/pkg/contract/interface.g.go

```diff
@@ -0,0 +1,1054 @@
+	TransitionModelVersionStage(input *protos.TransitionModelVersionStage) (*protos.TransitionModelVersionStage_Response, *MlflowError)
+	DeleteModelVersion(input *protos.DeleteModelVersion) (*protos.DeleteModelVersion_Response, *MlflowError)
+	GetModelVersion(input *protos.GetModelVersion) (*protos.GetModelVersion_Response, *MlflowError)
+	SearchModelVersions(input *protos.SearchModelVersions) (*protos.SearchModelVersions_Response, *MlflowError)
+	GetModelVersionDownloadUri(input *protos.GetModelVersionDownloadUri) (*protos.GetModelVersionDownloadUri_Response, *MlflowError)
+	SetRegisteredModelTag(input *protos.SetRegisteredModelTag) (*protos.SetRegisteredModelTag_Response, *MlflowError)
+	SetModelVersionTag(input *protos.SetModelVersionTag) (*protos.SetModelVersionTag_Response, *MlflowError)
+	DeleteRegisteredModelTag(input *protos.DeleteRegisteredModelTag) (*protos.DeleteRegisteredModelTag_Response, *MlflowError)
+	DeleteModelVersionTag(input *protos.DeleteModelVersionTag) (*protos.DeleteModelVersionTag_Response, *MlflowError)
+	SetRegisteredModelAlias(input *protos.SetRegisteredModelAlias) (*protos.SetRegisteredModelAlias_Response, *MlflowError)
+	DeleteRegisteredModelAlias(input *protos.DeleteRegisteredModelAlias) (*protos.DeleteRegisteredModelAlias_Response, *MlflowError)
+	GetModelVersionByAlias(input *protos.GetModelVersionByAlias) (*protos.GetModelVersionByAlias_Response, *MlflowError)
+}
+type MlflowArtifactsService interface {
+	Validate(input interface {
+	}) []string
+	DownloadArtifact(input *artifacts.DownloadArtifact) (*artifacts.DownloadArtifact_Response, *MlflowError)
+	UploadArtifact(input *artifacts.UploadArtifact) (*artifacts.UploadArtifact_Response, *MlflowError)
+	ListArtifacts(input *artifacts.ListArtifacts) (*artifacts.ListArtifacts_Response, *MlflowError)
+	DeleteArtifact(input *artifacts.DeleteArtifact) (*artifacts.DeleteArtifact_Response, *MlflowError)
+	CreateMultipartUpload(input *artifacts.CreateMultipartUpload) (*artifacts.CreateMultipartUpload_Response, *MlflowError)
+	CompleteMultipartUpload(input *artifacts.CompleteMultipartUpload) (*artifacts.CompleteMultipartUpload_Response, *MlflowError)
+	AbortMultipartUpload(input *artifacts.AbortMultipartUpload) (*artifacts.AbortMultipartUpload_Response, *MlflowError)
+}
+
+func RegisterMlflowServiceRoutes(service MlflowService, app *fiber.App) {
+	app.Get("/api/2.0/mlflow/experiments/get-by-name", func(ctx *fiber.Ctx) error {
+		input := &protos.GetExperimentByName{}
+		if err := ctx.QueryParser(input); err != nil {
+			return err
+		}
+		validationErrors := service.Validate(input)
+		if len(validationErrors) > 0 {
+			return &fiber.Error{Code: fiber.ErrBadRequest.Code, Message: strings.Join(validationErrors, " and ")}
+		}
+		output, err := service.GetExperimentByName(input)
+		if err != nil && err.ErrorCode == protos.ErrorCode_NOT_IMPLEMENTED {
+			return ctx.Next()
+		}
+		if err != nil {
+			return err
+		}
```

Comment:
this would be more idiomatic:
```suggestion
		if err != nil {
		    if err.ErrorCode == protos.ErrorCode_NOT_IMPLEMENTED {
			    return ctx.Next()
		    }
			return err
		}
```